### PR TITLE
Readme in sth package

### PR DESCRIPTION
**Pull request by a-tylenda**
what was done in STH readmes:
- typo corrections and duplicated text removal  ✍🏼 
- missing `readme.mtpl` in symbol package added 📝 
- not working links corrected 🔗 
- link to `package.json` replaced with an actual package.json content `{ }`
- badges and logo added
- new `README.md` for every package generated with command `node scripts/mk-readme.js`
- new readme-part `logo-header.md` added to folder `./conf/readme-parts/`

### CODE REVIEW

File: packages/adapters/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Scramjet Transform Hub Adapters
+ # Scramjet Transform Hub Adapters
 
 This module holds the docker adapters utilized by [Scramjet Transform Hub](https://github.com/scramjetorg/transform-hub).
```
Comment (alicja-gruzdz) [inline packages/adapters/README.md:3]: This module holds two types of adapters utilized by [Scramjet Transform Hub](https://github.com/scramjetorg/transform-hub): Instance Adapter and Sequence Adapter. These Adapters allows for running the Sequence identification and Instance execution in two basic modes: as a non containerized standalone processes or in a Docker container.

File: conf/readme-parts/links.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
 * The [Scramjet Framework - functional reactive stream processing framework](https://framework.scramjet.org)
-* The [Transform Hub repo onm github](https://github.com/scramjetorg/transform-hub)
-* You can see the [Scramjet Transform Hub API docs here](https://github.com/scramjetorg/transform-hub/tree/release/0.10/docs/development-guide/stream-and-api.md)
-* You can see the [CLI documentation here](https://github.com/scramjetorg/transform-hub/tree/release/0.10/docs/development-guide/scramjet-interface-cli.md), but `si help` should also be quite effective.
-* Don't forget to `star` this repo if you like it, `subscribe` to releases and keep visiting us for new versions and updates.
+* The [Transform Hub repo on github](https://github.com/scramjetorg/transform-hub)
+* You can see the [Scramjet Transform Hub API docs here](https://github.com/scramjetorg/transform-hub/tree/release/0.12/docs/development-guide/stream-and-api.md)
+* You can see the [CLI documentation here](https://github.com/scramjetorg/transform-hub/tree/release/0.12/docs/interfaces/CLI-command-reference.md), but `si help` should also be quite effective.
```
Comment (alicja-gruzdz) [inline conf/readme-parts/links.md:7]: @a-tylenda @agne6ka  are we're leaving both CLI/README and the interfaces/CLI-command-reference.md?

**Review by alicja-gruzdz — APPROVED** [review]
Excellent.
Much needed work :muscle:

File: packages/adapters/README.md (hunk 1 above)
Comment (a-tylenda) [reply packages/adapters/README.md:3]: Added to `readme.mtpl` in adapters package. Thanks Ala! nice catch ⚾

File: conf/readme-parts/links.md (hunk 2 above)
Comment (a-tylenda) [reply conf/readme-parts/links.md:7]: @alicja-gruzdz  @agne6ka  we can decide on this in PR [#142](https://github.com/scramjetorg/transform-hub/pull/142)

Comment (alicja-gruzdz) [reply conf/readme-parts/links.md]: in the meantime @a-tylenda and I decided that we will close the https://github.com/scramjetorg/transform-hub/pull/142 branch and make the CLI correction on this branch. As for the destiny of interfaces/CLI-command-reference.md it hasn't been decided yet. @agne6ka what ya think?

**Review by MichalCz — APPROVED** [review]
Nice, let's merge this and release.

**Pull request merged**